File: modules/Common.Logging/test/Common/Common.Logging.Integration.Tests.Vb/Class1.vb
Imports Common.Logging
Imports NUnit.Framework

<TestFixture()> _
Public Class Class1

    <Test()> _
    Public Sub CanCompile()
        Dim log As Common.Logging.ILog = LogManager.GetCurrentClassLogger()
        log.Trace(Function(m) m("test {0}", "test"))
    End Sub
End Class

File: modules/Common.Logging/test/Common/Common.Logging.Integration.Tests/Class1.cs
#if NET_3_0

using System;
using System.Globalization;
using Common.Logging;
using NUnit.Framework;

namespace Common
{
    /// <summary>
	/// Summary description for Class1.
	/// </summary>
	[TestFixture]
	public class Class1
	{
        [Test]
		public void CanCompile()
		{
            ILog log = LogManager.GetCurrentClassLogger();
            log.Trace(CultureInfo.InvariantCulture,  m => m("test {0}", "test"), new Exception());
		}
	}
}

#endif